dataflow/src/test/java/common/PerformDFATest.java

```diff
@@ -0,0 +1,38 @@
+package common;
+
+import org.checkerframework.dataflow.analysis.AbstractValue;
+import org.checkerframework.dataflow.analysis.Analysis;
+import org.checkerframework.dataflow.analysis.Store;
+import org.checkerframework.dataflow.analysis.TransferFunction;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizeLauncher;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class PerformDFATest {
```

Comment:
I think this feels like a name for a method instead of a class. Please try renaming it. My personal suggestion is `DFATester`.

Reply:
Please avoid the abbreviation "DFA". Simply use dataflow instead. Maybe `DataflowTester` is good.